test/pure/Map.test.mo

```diff
@@ -772,9 +772,10 @@ run(
       test(
         "repeated delete",
         do {
-          var rbMap = buildTestMap();
-          rbMap := Map.delete(rbMap, Nat.compare, 1);
-          Map.delete(rbMap, Nat.compare, 1)
+          let map = buildTestMap();
+          let (map1, true) = Map.delete(map, Nat.compare, 1);
```

Comment:
Maybe we could use an explicit `switch` expression to remove the compiler warnings?